code/part-one/README.md

```diff
@@ -187,12 +190,10 @@ Your blockchain will consist of three related ES6 classes:
 
 **Module:** [validation.js](validation.js)
 
-There are as many ways to validate a blockchain as there are blockchains. It is
-a huge and varied topic. In fact, in Sawtooth the central component is named a
-"validator", and that is the vast majority what it does. For this section
-though, you get to be your own validator, and finally verify all that
-cryptographic boilerplate you've been including with your blocks and
-transactions.
+Blockchain validation is a huge and varied topic. In Sawtooth, validation is so
+important that the central component is named a "validator". For this section,
+you get to be your own validator and finally verify all that cryptographic
```

Comment:
thoughts on `you get to be your own validator` => `you get to write your own validator`?

Reply:
I meant it to be a little casual and fun. Though I actually think it is more accurate than saying that are writing a validator. That implies there will be some sort of process. There are only going to be functions, which they would have to run themselves, making them the validator.